internal/acceptance_test/resource_host_test.go

```diff
@@ -139,12 +129,13 @@ func testWaitUntiHostReady(rsrc string) resource.TestCheckFunc {
 
 		hostState := rest.HOSTSTATE_NEW
 		for i := 0; i < hostStatePollCount && hostState != rest.HOSTSTATE_READY; i++ {
-			host, _, err := p.Client.HostsApi.GetByID(ctx, hostID)
+			time.Sleep(hostStateReadyWait)
+
+			host, resp, err := p.Client.HostsApi.GetByID(ctx, hostID)
 			if err != nil {
 				return fmt.Errorf("Host: %q not found: %s", hostID, err)
 			}
-
-			time.Sleep(hostStateReadyWait)
+			defer resp.Body.Close()
```

Comment:
Any reason to defer this instead of just calling it right away?  Deferring in a loop is iffy.  I don't see any use for resp.Body, should just close it right away